src/utils/notionHelpers.ts

```diff
@@ -0,0 +1,15 @@
+import { exampleConfiguration } from '../configuration';
+import { Notion } from '../services/Notion';
+
+/**
+ * Send user's data to Notion
+ *
+ * @param {string} text - Text for request to database
+ */
+export function sendDataToNotion(text: string): void {
+  const notion = new Notion();
+
+  notion.init(exampleConfiguration);
```

Comment:
This config is contained in the instance. So, why do you initiate your client every time?